### ISSUE TITLE
release file zip issue

### ISSUE THREAD

**Issue by Quentin-ctrL** [SpikeHimself/XPortal#37]
Hi,

I downloaded the release file from https://github.com/SpikeHimself/XPortal/releases/tag/v1.2.9
Extracting it makes a strange behaviour :
```
ls -lh 
total 200K
-rw-r--r-- 1 quentin quentin 3,6K 23 mars  09:12  CHANGELOG.md
-rw-r--r-- 1 quentin quentin  37K  1 févr. 13:11  icon.png
-rw-r--r-- 1 quentin quentin  414 23 mars  09:12  manifest.json
drwx------ 2 quentin quentin   40 23 mars  09:13 'plugins\XPortal\Translations\'
-rw-r--r-- 1 quentin quentin  307 31 janv. 14:02 'plugins\XPortal\Translations\Dutch\xportal-translations-dutch.json'
-rw-r--r-- 1 quentin quentin  303  8 févr. 22:46 'plugins\XPortal\Translations\English\xportal-translations-english.json'
-rw-r--r-- 1 quentin quentin  315  1 févr. 17:59 'plugins\XPortal\Translations\French\xportal-translations-french.json'
-rw-r--r-- 1 quentin quentin  317  2 févr. 19:18 'plugins\XPortal\Translations\German\xportal-translations-german.json'
-rw-r--r-- 1 quentin quentin  310  9 févr. 18:43 'plugins\XPortal\Translations\Korean\xportal-translations-korean.json'
-rw-r--r-- 1 quentin quentin  328  9 févr. 00:20 'plugins\XPortal\Translations\Polish\xportal-translations-polish.json'
-rw-r--r-- 1 quentin quentin  302  1 févr. 17:21 'plugins\XPortal\Translations\Portuguese_Brazilian\xportal-translations-portuguese_brazillian.json'
-rw-r--r-- 1 quentin quentin  418  9 févr. 00:20 'plugins\XPortal\Translations\Russian\xportal-translations-russian.json'
-rw-r--r-- 1 quentin quentin  312  3 févr. 23:18 'plugins\XPortal\Translations\Spanish\xportal-translations-spanish.json'
-rw-r--r-- 1 quentin quentin  42K 23 mars  09:13 'plugins\XPortal\XPortal.dll'
-rw-r--r-- 1 quentin quentin 7,6K 23 mars  09:12  README.md
-rw-r--r-- 1 quentin quentin  64K  2 avril 00:00  XPortal-v1.2.9.zip
```
It's the first time I see it but I searched and see that some windows tool are mixing up things with separators. 

I fixed path with this command  : 
```
for i in *; do new=${i//\\/\/}; newd=$(dirname "$new"); mkdir -p "$newd"; mv "$i" "$new"; done
```
https://unix.stackexchange.com/a/607751

After that, the plugin is working.

Can you fix the release to be compatible both on windows and unix ? 

Thanks

**Comment by SpikeHimself** [comment]
Hi, thanks for reporting!

This was actually reported a few days ago and has already been fixed. Just waiting on a few other things before I release a new version. I'll keep you updated here when that happens.

**Comment by SpikeHimself** [comment]
I have just released v1.2.10 in which this problem has been addressed.

**Comment by Quentin-ctrL** [comment]
I confirm it's fixed with the latest release :+1:
